public/app/features/templating/variable.ts

```diff
@@ -1,7 +1,9 @@
 import _ from 'lodash';
```

Comment:
Maybe this file could be split to variable_utils and types, as it's quite big?

Reply:
Great suggestion, I think that could be done in a future PR.

Reply:
added to #22564